src/frontend/src/lib/flows/authLastUsedFlow.svelte.ts

```diff
@@ -69,9 +69,7 @@ export class AuthLastUsedFlow {
         const jwt = await requestJWT(requestConfig, {
           nonce: get(sessionStore).nonce,
           mediation: "optional",
-          loginHint:
-            lastUsedIdentity.authMethod.openid.loginHint ??
-            lastUsedIdentity.authMethod.openid.sub,
+          loginHint: lastUsedIdentity.authMethod.openid.loginHint,
```

Comment:
The sub wasn't working with microsoft. It was putting that same sub in the email field, which was wrong and extremely confusing.